### PR TITLE
Setup and configure ESLint

### LINKED ISSUE #18: Setup and configure ESLint

### PR DESCRIPTION
**Pull request by troym9731**
Closes #18

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...0de6c4f641119826923547","results":{"filePath":"/Users/Troy/projects/contributions/azwebdevs.org/src/server/lib/perf/index.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/Troy/projects/contributions/azwebdevs.org/src/server/middleware/mongo/index.js":{"size":522,"mtime":1455356538000,"hashOfConfig":"4ed4b1591c0de6c4f641119826923547","results":{"filePath":"/Users/Troy/projects/contributions/azwebdevs.org/src/server/middleware/mongo/index.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/Troy/projects/contributions/azwebdevs.org/src/server/middleware/request-logger/index.js":{"size":856,"mtime":1455356538000,"hashOfConfig":"4ed4b1591c0de6c4f641119826923547","results":{"filePath":"/Users/Troy/projects/contributions/azwebdevs.org/src/server/middleware/request-logger/index.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/Troy/projects/contributions/azwebdevs.org/src/server/middleware/response-timer/index.js":{"size":178,"mtime":1455356538000,"hashOfConfig":"4ed4b1591c0de6c4f641119826923547","results":{"filePath":"/Users/Troy/projects/contributions/azwebdevs.org/src/server/middleware/response-timer/index.js","messages":[],"errorCount":0,"warningCount":0}}}
```
Comment (therealklanni) [inline .eslintcache]: We should probably gitignore this

File: package.json [hunk 2]
```diff
@@ -16,11 +16,6 @@
   },
   "author": "Kevin Lanni (https://github.com/therealklanni)",
   "license": "MIT",
-  "babel": {
```
Comment (therealklanni) [inline package.json:19]: Why was this removed?

File: .eslintcache (hunk 1 above)
Comment (troym9731) [inline .eslintcache]: Ah, it seems to specify a specific path on my computer. That's not good.

File: .babelrc [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (therealklanni) [inline .babelrc:1]: Oh, that's why

**Comment by therealklanni** [comment]
:+1:
